lib/busbar_cli/commands/logs.rb

```diff
@@ -3,43 +3,34 @@ module Logs
     extend ActiveSupport::Concern
 
     included do
-      desc 'logs APPLICATION_CONTAINER ENV [COMPONENT_TYPE]',
-        'Fetch the logs from a application container. You can find the container for your application through the "busbar containers" command.'
+      desc 'logs APP ENV [COMPONENT]',
+        'Fetch the logs from a application.'
       method_option :since,
                     default: '0',
                     type: :string,
                     desc: 'Only return logs newer than a relative duration like 5s, 2m, or 3h. ' \
                           'Defaults to all logs.'
       method_option :size,
-                    default: '100',
+                    default: '30',
                     type: :string,
                     desc: 'Last <size> lines of the logs to be returned' \
                           'Defaults to 100.'
```

Comment:
Please delete this line. Whatever CLI gem this app is using automatically translates line 14 (`default: '30'`) to the following output:

```sh
$ busbar help logs
Usage:
  busbar logs APPLICATION_CONTAINER ENV [COMPONENT_TYPE]

Options:
  [--since=SINCE]      # Only return logs newer than a relative duration like 5s, 2m, or 3h. Defaults to all logs.
                       # Default: 0
  [--size=SIZE]        # Last <size> lines of the logs to be returnedDefaults to 100. # ⬅️ this comes from this line that should be deleted
                       # Default: 100 # ⬅️ this comes from line 14 above
  [--profile=PROFILE]  # Profile used to run the command

Fetch the logs from a application container. You can find the container for your application through the "busbar containers" command.
```

Reply:
That is manually set and I forgot to set... fixing! =)

Reply:
Please delete this line. There is no need to repeat the default value twice.